### PR TITLE
feat(unified-share-modal): add external collab warning message

### PR DESCRIPTION
**Pull request by baipenghan**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @baipenghan has signed the CLA. Thanks for the pull request!

File: src/features/unified-share-modal/UnifiedShareModal.js [hunk 1]
```diff
@@ -367,16 +371,28 @@ class UnifiedShareModal extends React.Component<Props, State> {
         this.setState({ isEmailLinkSectionExpanded: false });
     };
 
+    hasExternalContact = (contacts: Array<Contact>) => {
+        return contacts.some(contact => contact.isExternalUser);
```
Comment (booc0mtaco) [inline src/features/unified-share-modal/UnifiedShareModal.js]: Please verify this works as expected in IE11

**Comment by cweeii** [comment]
Yes, the flowtype is updated here: src/features/unified-share-modal/flowTypes.js

File: src/features/unified-share-modal/UnifiedShareModal.js (hunk 1 above)
Comment (baipenghan) [reply src/features/unified-share-modal/UnifiedShareModal.js]: Good catch. I just verified on my Windows VM in IE11, and it works.

**Pull request merged**